chatkit-android/build.gradle

```diff
@@ -35,7 +51,7 @@ kotlin {
 dependencies {
     implementation "com.android.support:appcompat-v7:$android_support_version"
 
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
travis runs it on jdk8. is it compatible with 7? if it is we should probably test on 7...

Reply:
https://trello.com/c/gKBrewzX/924-test-android-sdk-with-jdk7